ctapipe/io/containers.py

```diff
@@ -75,6 +75,10 @@ class RawCameraContainer(Container):
     """
     Storage of raw data from a single telescope
     """
+    event_number = Item(-1, "telescope event number")
+    num_channels = Item(-1, "number of channels per pixel")
```

Comment:
num_channels and num_pixels are filled in the InstrumentContainer as these values do not change per event.

Event_id exists in RawDataContainer. Is event_number intended to be different to this value? 

pixel_flags is new to me, what is it's purpose? could you add a description?